src/main/java/com/thindeck/api/mock/MkItem.java

```diff
@@ -74,28 +80,47 @@ public MkItem(final long num) {
 
     @Override
     public AttributeValue get(final String str) throws IOException {
-        throw new UnsupportedOperationException("#get");
+        AttributeValue value = null;
```

Comment:
if we won't find attribute value then just throw `IllegalArgumentException`, we shouldn't return nulls anywhere